deleteme

```diff
@@ -0,0 +1 @@
+foo
```

Comment:
Delete me? 😉

Reply:
It's in my --wip-- commit. Will vanish when I squash up ;)